### PR TITLE
Added Lifesize Cloud 10.3.2

### PR DESCRIPTION
**Pull request by guandalf**
### Checklist
- [X] The commit message includes the cask’s name and version.
- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` left no offenses.

Additionally, when **adding a new cask**:
- [X] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [X] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [X] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/lifesize.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask 'lifesize' do
+  version '10.3.2'
+  sha256 '5661cc5e78efad1b1c3c5a4b3d3635e2c2514e2ba87335362c83677cbd3e12ee'
+
+  # cdn.lifesizecloud.com was verified as official when first introduced to the cask
+  url 'http://cdn.lifesizecloud.com/LifesizeCloud-10.3.2-181-signed.pkg'
```
Comment (vitorgalvao) [inline Casks/lifesize.rb]: I can’t find the download link, except for being linked to a registration wall. Is there a public accessible page with the link?

File: Casks/lifesize.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+cask 'lifesize' do
+  version '10.3.2'
```
Comment (vitorgalvao) [inline Casks/lifesize.rb]: `version '10.3.2-181'`

File: Casks/lifesize.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/lifesize.rb]: `url "http://cdn.lifesizecloud.com/LifesizeCloud-#{version}-signed.pkg"`

File: Casks/lifesize.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+  # cdn.lifesizecloud.com was verified as official when first introduced to the cask
+  url 'http://cdn.lifesizecloud.com/LifesizeCloud-10.3.2-181-signed.pkg'
+  name 'lifesize'
+  homepage 'https://www.lifesize.com/'
+  license :commercial
+
+  auto_updates true
+
+  pkg 'LifesizeCloud-10.3.2-181-signed.pkg'
```
Comment (vitorgalvao) [inline Casks/lifesize.rb]: `pkg "LifesizeCloud-#{version}-signed.pkg"`

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/lifesize.rb (hunk 1 above)
Comment (guandalf) [inline Casks/lifesize.rb]: I've got it from here, no login or registration with FF in private mode:
[https://call.lifesizecloud.com/download]

Comment (vitorgalvao) [inline Casks/lifesize.rb]: Confirmed. Please make the other fixes, and we can merge this.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**